spec/features/language_helper_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'receive_correct_diapers_form' do
+  context 'float numbers' do
+    it 'returns correct word form for float number' do
+      helper = LanguageHelper::UkrLanguageHelper.new
+    expect(helper.corr_word_form(421.0)).to eql('підгузок')
+    expect(helper.corr_word_form(34711.15)).to eql('підгузків')
```

Comment:
use let to declare parameter variable inside  corr_word_form

Reply:
fixed